### DISCUSSION TITLE
[vite-plugin-global-style] Nuxt 3 Support and Integration

### DISCUSSION THREAD

**Discussion by ijkml** [originjs/origin.js#155] [Q&A]
Is there currently a way to add this plugin to Nuxt 3? If there isn't, do you plan to make it available?
Thanks.

**Comment by PeterAlfredLee** [comment]
Hi @ijkml 
I only have limited understanding about Nuxt. Is vite or rollup interated in Nuxt 3?
If so, I believe `vite-plugin-global-style` can be used then.

**Reply by ijkml** [reply]
Hey...

I actually use another way, but yes it can be set up as a Vite plugin 💚.
